sdks/go/pkg/beam/core/util/reflectx/functions_test.go

```diff
@@ -41,3 +45,19 @@ func TestLoadFunction(t *testing.T) {
 		t.Errorf("got %d, wanted %d", out[0].Int(), testFunction())
 	}
 }
+
+func TestFunctionOutputSize(t *testing.T) {
+	expected := 1
+	received := FunctionOutputSize(testFunction)
+	if received != expected {
+		t.Errorf("got %d, wanted %d", received, expected)
+	}
+}
+
+func TestFunction2OutputSize(t *testing.T) {
+	expected := 2
+	received := FunctionOutputSize(testFunction2)
+	if received != expected {
+		t.Errorf("got %d, wanted %d", received, expected)
```

Comment:
It should be possible for a user to not look at the test code to understand what happened for this test.

So a better test output would be:
t.Errorf("FunctionOutputSize(%v) =  %d, want %d", <testfunctionname/signature>,received, expected)

Reply:
945dc7a264